testing/constraints-3.8.txt

```
@@ -1 +1,13 @@
-sqlalchemy==1.3.24
+# This constraints file is used to check that lower bounds
+# are correct in setup.py
+# List *all* library dependencies and extras in this file.
+# Pin the version to the lower bound.
+#
+# e.g., if setup.py has "foo >= 1.14.0, < 2.0.0dev",
+sqlalchemy>=1.4.0
```

You should pick an exact version, otherwise we get the latest compatible.

```suggestion
sqlalchemy==1.4.0
```

Ideally this would reflect the minimum version we support.